# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/CompilationInfo.h
# Commit: 3749d6d2325d
# Full Hash: 3749d6d2325d36db3987be586fcfba07b9423356
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 7: Add CompilationStencil::getParserAtomAt. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96914
# ==============================================================================

diff -r 50e9d084764c -r 3749d6d2325d js/src/frontend/CompilationInfo.h
--- a/js/src/frontend/CompilationInfo.h	Sat Nov 14 00:25:21 2020 +0000
+++ b/js/src/frontend/CompilationInfo.h	Sat Nov 14 00:03:00 2020 +0000
@@ -287,6 +287,9 @@
     alloc.steal(&other.alloc);
   }
 
+  const ParserAtom* getParserAtomAt(JSContext* cx,
+                                    TaggedParserAtomIndex taggedIndex) const;
+
 #if defined(DEBUG) || defined(JS_JITSPEW)
   void dump();
   void dump(js::JSONPrinter& json);